fuzzing/ossfuzz.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash -eu
+
+# This script is meant to be run by
+# https://github.com/google/oss-fuzz/blob/master/projects/cjson/Dockerfile
+
+mkdir build
+cd build
```

Comment:
suggest a judgement, like:
```
mkdir -p libfuzzer-build || exit 1
cd libfuzzer-build || exit 1
```

Reply:
That is supposed to be run in a OSS-Fuzz environment where those directories always have to be created, it's not useful outside that environment.